bigframes/core/compile/aggregate_compiler.py

```diff
@@ -431,6 +431,19 @@ def _(
     return cast(ibis_types.NumericColumn, bq_corr)
 
 
+@compile_binary_agg.register
+def _(
+    op: agg_ops.CovOp, left: ibis_types.Column, right: ibis_types.Column, window=None
+) -> ibis_types.NumericValue:
+    # Will be null if all inputs are null. Pandas defaults to zero sum though.
+    left_numeric = cast(ibis_types.NumericColumn, left)
+    right_numeric = cast(ibis_types.NumericColumn, right)
+    bq_cov = _apply_window_if_present(
+        left_numeric.cov(right_numeric, how="sample"), window
```

Comment:
what's `how='sample'` meaning here? Do we calculate the `cov` based on a sample series?

Reply:
Sample variance often means it is population_variance * N / N-1. This provides an unbiased estimator of variance https://mathworld.wolfram.com/SampleVariance.html.